### PR TITLE
Add load testing as a manual step. Locust tool

### PR DESCRIPTION
**Pull request by rgparkins**
Added a github action to be run manually. This tests against the development environment and as such would need the github runner IP address range applied to that environment for access

### CODE REVIEW

File: .github/workflows/load-testing.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Locust oad testing
```
Comment (albal) [inline .github/workflows/load-testing.yml:1]: Spelling

Comment (rgparkins) [reply .github/workflows/load-testing.yml]: Corrected

**Comment by rgparkins** [comment]
I have added a readme and also removed the tools directory

**Comment by albal** [comment]
You need to add the following to your action or else it will run on every push, merge etc.:

on:
  workflow_dispatch:
  
See the app-deploy action for more info.

**Comment by rgparkins** [comment]
> You need to add the following to your action or else it will run on every push, merge etc.:
> 
> on: workflow_dispatch:
> 
> See the app-deploy action for more info.

Take a look @albal I think Ive sorted

**Pull request merged**